src/Sylius/Component/Translation/spec/Fixtures/SampleTranslatableResource.php

```diff
@@ -11,11 +11,13 @@
 
 namespace spec\Sylius\Component\Translation\Fixtures;
 
-use Sylius\Component\Translation\Model\AbstractTranslatable;
+use Sylius\Component\Translation\Model\TranslatableInterface;
+use Sylius\Component\Translation\Model\TranslatableTrait;
 
 /**
  * @author Paweł Jędrzejewski <pawel@sylius.org>
  */
-class SampleTranslatableResource extends AbstractTranslatable
+class SampleTranslatableResource implements TranslatableInterface
```

Comment:
Personally I think leaving the interface would ease any case where we need to know if a model is translatable or not. Checking by traits is a little hacky at the moment, especially when using [reflections](http://php.net/manual/pl/class.reflectionclass.php).